src/api.js

```diff
@@ -15,16 +17,29 @@ window.api = {
   doExport() {
```

Comment:
I'm not sure that this style of function def is well-supported. Looks like it's good in Chrome and Firefox, but not in Safari. Is that a concern?

Specifically I'm talking about using 

```
{
   doExport(data) {
        //...
    }
}
```
as a shorthand for
```
{
    doImport: function(data) {
        //...
    }
}
```